frontend/server/src/Request.php

```diff
@@ -476,6 +476,49 @@ public function ensureMainUserIdentity(): void {
         }
     }
 
+     /**
+     * Ensures that an identity is logged in is Over 13 years of age.
+     *
+     * @throws \OmegaUp\Exceptions\UnauthorizedException
+     * @psalm-assert !null $this->identity
+     * @psalm-assert !null $this->identity->identity_id
+     * @psalm-assert !null $this->identity->username
+     */
+    public function ensureIdentityIsOver13(): void {
+        $this->ensureIdentity();
+        if (is_null($this->user)) {
+            return;
+        }
+        if (\OmegaUp\Authorization::isUnderThirteenUser($this->user)) {
+            throw new \OmegaUp\Exceptions\ForbiddenAccessException(
+                'U13CannotPerform' // A translation string for users who can not perform this action
```

Comment:
```suggestion
                'U13CannotPerform'
```

no need to explain this code. its contents have sufficient explanation.

Reply:
okayy